plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -411,21 +410,19 @@ export class PersonState {
         mergeIntoDistinctId,
         otherPerson,
         otherPersonDistinctId,
-        shouldIdentifyPerson = true,
     }: {
         mergeInto: Person
         mergeIntoDistinctId: string
         otherPerson: Person
         otherPersonDistinctId: string
-        shouldIdentifyPerson?: boolean
     }): Promise<void> {
         const olderCreatedAt = DateTime.min(mergeInto.created_at, otherPerson.created_at)
         const newerCreatedAt = DateTime.max(mergeInto.created_at, otherPerson.created_at)
 
         const mergeAllowed = this.isMergeAllowed(otherPerson)
 
         this.statsd?.increment('merge_users', {
-            call: shouldIdentifyPerson ? 'identify' : 'alias',
+            call: this.event.event, // $identify, $create_alias or $merge_dangerously
```

Comment:
Do we use this merge_users? and do we care about keeping it backwards compat with the old values?

Reply:
no-one besides me looks at this, so we're good.